packages/core/contracts/data-verification-mechanism/implementation/VotingV2.sol

```diff
@@ -651,6 +666,17 @@ contract VotingV2 is Staker, OracleInterface, OracleAncillaryInterface, OracleGo
         emit MaxRollsChanged(newMaxRolls);
     }
 
+    /**
+     * @notice Sets the maximum number of requests that can be made in a single round. Used to bound the maximum
+     * sequential slashing that can be applied within a single round.
+     * @dev Can only be called by the contract owner.
+     * @param newMaxRequestsPerRound the new maximum number of requests that can be made in a single round.
+     */
+    function setMaxRequestPerRound(uint32 newMaxRequestsPerRound) public override onlyOwner {
+        maxRequestsPerRound = newMaxRequestsPerRound;
```

Comment:
shall we require >0 in order to avoid setting where no requests can be processed?

Reply:
good call! will add this restriction now